content/english/articles/effective-gnops/onboarding-testnet-validator/index.md

```diff
@@ -1,11 +1,85 @@
 ---
 title: Onboarding a testnet validator
-publishDate: 2025-03-11T08:00:00-01:00
+publishDate: 2025-03-14T08:00:00-01:00
 translationKey: "bootnodes-persistent-peers"
-tags: [ "validator", "testnet", "onboarding" ]
+tags: ["validator", "valopers", "testnet", "onboarding"]
 level: Intermediate
-author: None
-summary: "This give a set of information on onboarding a validator node: how to provision the right server and connect it to the testnet"
+author: d4ryl00
+summary: "This give a set of information on onboarding a validator node: how to register as a validator operator (valoper) in the registry and how to submit a validator proposal"
 ---
 
-## TBD
+# Setting Up a Validator Node in gno.land
+
+## Overview
+
+Validators on **gno.land** are expected to demonstrate their technical expertise and alignment with the project by making continuous and meaningful contributions. **gno.land** abstracts validator management into the `r/sys/vals` realm as a form of a smart contract for modularity.
+
+This guide walks you through the process of registering your validator node into the validator set with a smart-contract. It assumes that you already have an operational validator node running on the testnet. If not, follow [this guide](https://gnops.io/articles/guides/remote-chain/).
+
+## Step 1: Creating a `Valoper` object
+
+To be added to the validator set, you must create a **Valoper** object and register it in the `r/gnoland/valopers` realm using the `Register` function. This object allows you to demonstrate to **GovDAO members** why you should be accepted as a validator.
```

Comment:
`To be considered to be added to the validator set`

Reply:
done in https://github.com/gnoverse/gnops/pull/27/commits/6f4e1f77b426dc6f69c79fd9afd974e064d944f4